packages/aws-cdk-lib/aws-ec2/lib/vpc-endpoint.ts

```diff
@@ -396,42 +423,60 @@ export class InterfaceVpcEndpointAwsService implements IInterfaceVpcEndpointServ
-  public static readonly PROMETHEUS = new InterfaceVpcEndpointAwsService('aps');
-  public static readonly PROMETHEUS_WORKSPACES = new InterfaceVpcEndpointAwsService('aps-workspaces');
+  public static readonly MACIE = new InterfaceVpcEndpointAwsService('macie2');
+  public static readonly MANAGEMENT_CONSOLE = new InterfaceVpcEndpointAwsService('console');
+  public static readonly MANAGEMENT_CONSOLE_SIGNIN = new InterfaceVpcEndpointAwsService('signin');
+  public static readonly MANAGED_BLOCKCHAIN_QUERY = new InterfaceVpcEndpointAwsService('managedblockchain-query');
+  public static readonly MANAGED_BLOCKCHAIN_BITCOIN_MAINNET = new InterfaceVpcEndpointAwsService('managedblockchain.bitcoin.mainnet');
+  public static readonly MANAGED_BLOCKCHAIN_BITCOIN_TESTNET = new InterfaceVpcEndpointAwsService('managedblockchain.bitcoin.testnet');
   public static readonly MEMORY_DB = new InterfaceVpcEndpointAwsService('memory-db');
   public static readonly MEMORY_DB_FIPS = new InterfaceVpcEndpointAwsService('memorydb-fips');
   public static readonly MIGRATIONHUB_ORCHESTRATOR = new InterfaceVpcEndpointAwsService('migrationhub-orchestrator');
   public static readonly MIGRATIONHUB_REFACTOR_SPACES = new InterfaceVpcEndpointAwsService('refactor-spaces');
   public static readonly MIGRATIONHUB_STRATEGY = new InterfaceVpcEndpointAwsService('migrationhub-strategy');
+  public static readonly NEPTUNE_ANALYTICS = new InterfaceVpcEndpointAwsService('neptune-graph');
   public static readonly NIMBLE_STUDIO = new InterfaceVpcEndpointAwsService('nimble');
+  public static readonly ORGANIZATIONS = new InterfaceVpcEndpointAwsService('organizations');
+  public static readonly ORGANIZATIONS_FIPS = new InterfaceVpcEndpointAwsService('organizations-fips');
   public static readonly OMICS_ANALYTICS = new InterfaceVpcEndpointAwsService('analytics-omics');
   public static readonly OMICS_CONTROL_STORAGE = new InterfaceVpcEndpointAwsService('control-storage-omics');
   public static readonly OMICS_STORAGE = new InterfaceVpcEndpointAwsService('storage-omics');
   public static readonly OMICS_TAGS = new InterfaceVpcEndpointAwsService('tags-omics');
   public static readonly OMICS_WORKFLOWS = new InterfaceVpcEndpointAwsService('workflows-omics');
   public static readonly PANORAMA = new InterfaceVpcEndpointAwsService('panorama');
+  public static readonly PAYMENT_CRYPTOGRAPHY_CONTROLPLANE = new InterfaceVpcEndpointAwsService('payment-cryptography.controlplane');
+  public static readonly PAYMENT_CRYTOGRAPHY_DATAPLANE = new InterfaceVpcEndpointAwsService('payment-cryptography.dataplane');
+  public static readonly PERSONALIZE = new InterfaceVpcEndpointAwsService('personalize');
+  public static readonly PERSONALIZE_EVENTS = new InterfaceVpcEndpointAwsService('personalize-events');
+  public static readonly PERSONALIZE_RUNTIME = new InterfaceVpcEndpointAwsService('personalize-runtime');
+  public static readonly PINPOINT_V1 = new InterfaceVpcEndpointAwsService('pinpoint');
```

Comment:
I don't like this name either but I don't have much choice.

Reply:
Same here, deprecate `PINPOINT`, and point to a correctly renamed `PINPOINT_SMS_VOICE_V2`

Reply:
But then call `InterfaceVpcEndpointAwsService('pinpoint');` what?

Reply:
I guess `PINPOINT_V1` or something similar, but with both `PINPOINT` and `PINPOINT_V1` being valid, I would expect `PINPOINT` to be `InterfaceVpcEndpointAwsService('pinpoint');`. I feel like also renaming it would lower the chances of confusion.

Reply:
let me know what you think of that...

Reply:
Yeah I think this is the solution less likely to cause pitfalls for future users, even if it doesn't feel great. Thanks 👍